Source/WebKit/Shared/WebEvent.h

```diff
@@ -71,12 +71,12 @@ class WebEvent {
     WebEvent();
 
 private:
-    WebEventType m_type;
+    std::optional<WebEventType> m_type;
```

Comment:
This shouldn't be std::optional, using the diff I sent you offline.  That will remove a lot of std::optional's.